eth-contracts/contracts/service/DelegateManager.sol

```diff
@@ -9,10 +9,9 @@ import "./ServiceProviderFactory.sol";
 import "./ClaimsManager.sol";
 import "../InitializableV2.sol";
 
-/**
- * Designed to manage delegation to staking contract
- * @notice - will call RegistryContract.constructor, which calls Ownable constructor
- */
+
+ /// Designed to manage delegation to staking contract
+ /// @notice - will call RegistryContract.constructor, which calls Ownable constructor
```

Comment:
why change from /** *? to ///?

Reply:
https://solidity.readthedocs.io/en/v0.5.10/natspec-format.html